fusequery/query/src/sql/plan_parser.rs

```diff
@@ -447,6 +448,8 @@ impl PlanParser {
         // In example: Aliases=[("id", (number % 3))]
         let aliases = extract_aliases(&projection_exprs);
 
+        let projection_exprs = RewriteHelper::rewrite_projection_aliases(&projection_exprs)?;
```

Comment:
Actually, I decide to remove rewrite_projection_aliases, because we choose to mysql alias style rather than clickhouse alias style.

Reply:
Got it. Should I close this PR?